packages/teleport-plugin-vue-base-component/src/index.ts

```diff
@@ -52,7 +53,8 @@ export const createVueComponentPlugin: ComponentPluginFactory<VueComponentConfig
           const iterator = useIndex ? `(${iteratorName}, index)` : iteratorName
           return `${iterator} in ${iteratedCollection}`
         },
-        customElementTagName: (value) => value,
+        customElementTagName: (value) => UIDLUtils.createWebComponentFriendlyName(value),
+        customElementImportName: (value) => `App${value}`,
```

Comment:
I wouldn't create a separate import name as the two need to be in sync to work

Reply:
what I mean is that you can't have two different functions, the difference between the tag name and the import name is the camel case or dash case